cartography/rules/spec/model.py

```diff
@@ -0,0 +1,77 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import Any
+
+
+class Provider(str, Enum):
```

Comment:
I think for the sake of consistency, we should call this "Module" given that's what we've called various cartography integrations since inception